blurring_as_a_service/performace_evaluation_pipeline/source/evaluate_with_coco.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from pycocotools.coco import COCO
+
+from metrics.customCocoEval import CustomCOCOeval
+
+
+def coco_evaluation(annotations_json, yolo_output_folder):
+
+    predictions_json = (
+        f"{yolo_output_folder}/exp/last-purple_boot_3l6p24vb_predictions.json"
+    )
+    anno = COCO(annotations_json)  # init annotations api
```

Comment:
Not a big fan in general of shortening the names since the editors come with autocomplete.
So I would prefer annotations instead of anno for clarity.